src/roman.rs

```diff
@@ -1,6 +1,6 @@
+use std::{fmt::{self, Display}, num::NonZeroU16, str::FromStr};
```

Comment:
This is purely subjective, but you and me just don't agree on how this sort of thing ought to be styled. :)

Reply:
No problem, which style do you prefer?

Reply:
I like having the crate includes at the top of the list rather than off by themselves. Not a big deal. Don't worry about it. :)